File: rb/test/exists_test.rb
# GrassTouch SDK exists test

require "minitest/autorun"
require_relative "../GrassTouch_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = GrassTouchSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- GrassTouch SDK exists test

local sdk = require("grass-touch_sdk")

describe("GrassTouchSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
